tests/testthat/test-result.R

```diff
@@ -45,7 +45,7 @@ test_that("inherited serialization/deserialization methods work", {
 
 test_that("S3 methods work", {
```

Comment:
wanna add a test if the `benchmark` object is malformed? there are a lot of rules there

Reply:
yeah, a good idea. it's overkill really; `numeric_version()` already does a lot of checking for us